probe-rs/src/probe/stlink/mod.rs

```diff
@@ -1242,31 +1248,58 @@ impl<D: StLinkUsb> SwoAccess for StLink<D> {
     }
 }
 
+/// ST-Link specific errors.
 #[derive(thiserror::Error, Debug)]
-pub(crate) enum StlinkError {
+pub enum StlinkError {
+    /// Invalid voltage values returned by probe.
```

Comment:
Do we wanna introduce https://docs.rs/displaydoc/latest/displaydoc/ while we are at it? :)

Reply:
Not really 😅

Reply:
oh I missed that this works with thiserror... well then maybe in another PR before this rots away